integration-tests/groovy-dsl/pom.xml

```diff
@@ -39,6 +39,26 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-direct</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-seda</artifactId>
```

Comment:
Please can you make sure these `org.apache.camel-quarkus` dependencies are also in the `virtualDependencies` profile (you can add them automatically with `mvn process-resources -Pformat -N`).

Reply:
It should be good now, thx